py3status/modules/window_title.py

```diff
@@ -51,7 +51,12 @@ def __init__(self):
     def window_title(self):
         window = find_focused(i3.get_tree())
 
+        if not window or window.get('name') is None or \
+           str(window.get('num')) == window.get('name'):
```

Comment:
>  I see scratchpad workspace along with window_title displaying 0:scratchpad.
Thanks for mentioning that.  I rolled back my solution to my first idea to just check num attr.

And it looks like empty workspace is the only condiition when window has num attribute.